### ISSUE TITLE
custom data test

### ISSUE THREAD

**Issue by wolf943134497** [W-Ted/N3D-VLM#3]
Hi @W-Ted Thanks for your great works！

Using custom data from a living room scene for model inference yields poor results. I think N3D-VLM have zero-shot capabilities for living room scenes, as the training data includes many living room scene samples. Do you have any suggestions?
Thanks！

**Comment by W-Ted** [comment]
Hi @wolf943134497, our inference pipeline will get 3D grounding results first, and then do 3D reasoning based on the grounding results. For the custom image, you may make the image path and the question in the format shown in [demo.json](https://github.com/W-Ted/N3D-VLM/blob/main/config/demo.json), and put images in the corresponding path. 

If you find that the 3D grounding result of your custom scene is abnormal, could you share your data? I can take a look of it.

**Comment by wolf943134497** [comment]
@W-Ted  my test data: 
I want to do the 3D Detection / Grouding only, air conditioner, soft, coffee table, tv ……
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/75d8bdf9-2e81-4aff-8b28-1692057a63c1" />
my test results:

<img width="832" height="549" alt="Image" src="https://github.com/user-attachments/assets/785139c1-7f26-442f-bf87-a996f7ac433f" />

**Comment by W-Ted** [comment]
Hi @wolf943134497, I tried your example, and get the result as below. I will update with an inference script for 3D detection only later. Please note that our current version of the model outputs bounding boxes in the AABB format for object localization.

<img width="1598" height="1038" alt="Image" src="https://github.com/user-attachments/assets/6a18941d-f4bc-4239-9217-1440b1ecd697" />

**Comment by wolf943134497** [comment]
Thank you for your response. It appears that the orientations of the 3D bbox have significant deviations, and there are also some discrepancies in their sizes. Could this be due to differences between the training data and the test data? The performance seems poorer than open-vocabulary monocular 3D detection methods that rely on camera intrinsics.

**Comment by W-Ted** [comment]
Regarding the orientation deviations, we would like to clarify that our current model is intentionally designed to output Axis-Aligned Bounding Boxes (AABB) rather than Oriented Bounding Boxes (OBB). This is a strategic choice within our current data and training pipeline.

While our data generation framework is extensive and scalable, this initial version prioritizes the scaling of data volume and the capabilities of reasoning. Consequently, we temporarily simplified the orientation annotations to AABB. This design choice naturally results in the visual discrepancies observed when comparing against OBBs for 3D pose estimation.

**Comment by wolf943134497** [comment]
got it. If orientation annotation information is incorporated into the retraining of the model, it should resolve the orientation deviation issue, right? Will the training code be made available in the future?

another question，since Qwen3-VL possesses certain 3D detection capabilities, why wasn't it built upon the Qwen3-VL model? That would likely improve the performance.

**Comment by wolf943134497** [comment]
I have tested the newly updated code, and the results meet expectations. Thanks
